build/restore.ps1

```diff
@@ -1,7 +1,14 @@
 $root = $PSScriptRoot
+$packages = "$root\..\packages"
+
 $nuget = & "$root\Get-Nuget.ps1"
 
-& $nuget restore $root\..\PortabilityTools.sln
+if (!(Test-Path $packages))
```

Comment:
Can you add a comment as to why we're doing this? Otherwise, we might remove it at some point, and then the tests start failing in automated builds